.gitignore

```diff
@@ -1,2 +1,9 @@
 libs
 test-reports
+
```

Comment:
You should add this to your global `gitignore` file.

```git config --global core.excludesfile <path-to-global-gitignore-file>```

Reply:
OK! I reverted it!